src/pages/gd/Claim/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { memo } from 'react'
+import { t } from '@lingui/macro'
+import { useLingui } from '@lingui/react'
+import { ClaimButton, Layout, Title } from '@gooddollar/good-design'
+import { StyleSheet } from 'react-native'
+import { View, Text } from 'native-base'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+
+const styles = StyleSheet.create({
```

Comment:
protocol ui app uses different approach to style. please look at another components

Reply:
@roma-hladilka ^^^

Reply:
done